massa-ledger-worker/src/ledger.rs

```diff
@@ -106,9 +107,16 @@ impl LedgerController for FinalLedger {
     ///
     /// # Returns
     /// A copy of the found bytecode, or None if the ledger entry was not found
-    fn get_bytecode(&self, addr: &Address) -> Option<Vec<u8>> {
+    fn get_bytecode(&self, addr: &Address) -> Option<Bytecode> {
+        let bytecode_deserializer = BytecodeDeserializer::new(u64::MAX);
```

Comment:
We are currently using the `max_datastore_value_length` for the max size of the bytecode also (maybe add 4 for the version bytes)

Reply:
You should use the one in self.config so that we can change it in tests etc

Reply:
? self.config (a LedgerConfig) does not have the max_datastore_value_length field right now. Only max_key_length and max_ledger_part_size.
Should I change the LedgerConfig to have this field available?

Reply:
Yes we avoid using the constants directly (sorry was thinking it was already in the struct)

Reply:
Resolved by https://github.com/massalabs/massa/pull/3603/commits/80e887d469b085f41d9f4f67cb363ab6fbfc74a4